next/src/components/global/Header/Header.module.scss

```diff
@@ -0,0 +1,28 @@
+.skipToMainContent {
+  z-index: 45;
+  opacity: 0;
+  pointer-events: none;
+  position: fixed;
+  left: 21px;
+  top: 21px;
+  overflow: hidden;
+  padding: 13px 21px;
+  background-color: var(--neutral-100, #fcfcfc);
+  font-weight: 700;
+  text-transform: uppercase;
+
+  &:focus-visible {
+    opacity: 1;
+    pointer-events: auto;
+  }
+}
+
+.header {
+  position: sticky;
+  top: 0;
+  left: 0;
+  width: 100vw;
```

Comment:
I would not add `width` for that. The `block` displayed elements have always 100% width by default, and the `100vw` can cause some troubles sometimes, for example with horizontal scrolling.